### DISCUSSION TITLE
Setting number of quadrature points

### DISCUSSION THREAD

**Discussion by tairoon1** [idaholab/moose#16549] [Q&A Modules: General] [answered]
Hi,

I was wondering, how can I set the number of quadrature points used? I am using a QUAD8 mesh and would like to use 2 quadrature points in each direction. The default seems to be three. Can I choose the number in the input file?

**Comment by hugary1995** [comment, marked as answer]
Look at these tests: [https://github.com/idaholab/moose/tree/next/test/tests/quadrature](https://github.com/idaholab/moose/tree/next/test/tests/quadrature) 

You can control the order of quadrature.

**Reply by tairoon1** [reply]
perfect, thanks!
